src/main/java/stsjorbsmod/cards/CustomJorbsModCard.java

```diff
@@ -238,4 +237,11 @@ public void triggerOnGlowCheck() {
         }
         return null;
     }
+
+    @Override
+    public void receiveStartAct() {
```

Comment:
For a case like this where there is base class functionality we want to always happen and also we want to make the same hook something that cards like Equinox can override, I'd recommend a pattern like this:

```java
// This is what JorbsMod::receiveStartAct calls; it is final, so individual cards can't override it
public final void receiveStartAct() {
    // <base class exerted functionality>
    this.atStartOfAct();
}
// Individual cards override this (and don't have to remember to call super.sameMethod() because of the pattern)
protected void atStartOfAct() { }
```

Reply:
Oh, oops, except that we shouldn't be using `CustomJorbsModCard` for this particular bit of functionality, because it needs to apply to base game cards that get exerted as well